src/garage/sampler/__init__.py

```diff
@@ -18,7 +19,7 @@
 
 __all__ = [
     'BatchSampler', 'Sampler', 'ISSampler', 'singleton_pool', 'LocalSampler',
-    'RaySampler', 'RL2Sampler', 'SamplerWorker', 'ParallelVecEnvExecutor',
-    'VecEnvExecutor', 'OffPolicyVectorizedSampler',
+    'RaySampler', 'RL2Sampler', 'RL2Worker', 'SamplerWorker',
```

Comment:
@krzentner do you think this kind of stuff should go in the algorithm or in the common directories like this? i think algorithm.

Reply:
if KR doesn't answer in time, please just put it inside the RL2 module, just like you put RL2Env there.

Reply:
Agree

Reply:
the whole idea of the worker interface is that algorithms can define their own workers, which are unique, and use them with this package, which is shared.

shared stuff goes here.

unique stuff should go in the algorithm.

Reply:
So far in garage, our algorithms don't have directories of their own. You could stick the classes the algorithm needs in that algorithm's file, or we could start using algorithm directories (like in rlkit). I think it would look weird for this to be the only algorithm with its own directory, but it would also be weird to place the class in the main algorithm's file.

I think it's fine where it is now, but at some point we do need to think more carefully about what we want to expose in the common directory and how we want to organize algorithm details like this.

Reply:
Let RL2 be the first one?